x-pack/plugins/security_solution/public/common/components/links_to_docs/external_link.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiLink } from '@elastic/eui';
+
+interface ExternalLinkProps {
+  url: string;
+  text: string;
+  ariaLabel?: string;
+}
+
+/**
+ * A simplistic text link for opening external urls in a new browser tab.
+ */
+export const ExternalLink: FC<ExternalLinkProps> = ({ url, text, ariaLabel }) => {
+  return (
+    <EuiLink href={url} aria-label={ariaLabel} external target="_blank" rel="noopener">
+      {text}
+    </EuiLink>
+  );
+};
```

Comment:
I think that there's already an `ExternalLink` component found here - `x-pack/plugins/security_solution/public/common/components/links/index.tsx` - we may want to make use of that one, or move this one into that file?

Reply:
Thank you, this file seemed too busy for me and containing specific implementations like `HostDetailsLinkComponent`, so I overlooked `ExternalLink` there.

I'll check how it works and looks with the `ExternalLink` component from `x-pack/plugins/security_solution/public/common/components/links/index.tsx` and try to submit some quick improvement if possible.